pre_commit_hooks/private_key_check.py

```diff
@@ -2,7 +2,7 @@
 import sys
 from typing import Optional
 
-REGEX_PATTERN = r'(?i)(?:"|\')(?:0x|[0-9a-f]{2})(?:[a-f0-9]{64}|[a-f0-9]{62})(?:"|\')(?! *# *noqa:keycheck|.*#noqa:keycheck)'
+REGEX_PATTERN = r'(?i)(?:"|\')(?:0x|[0-9a-f]{2})(?:[a-f0-9]{64}|[a-f0-9]{62})(?:"|\')(?:[,])(?! *# *noqa:keycheck|.*#noqa:keycheck)'
```

Comment:
Can you catch any kind of symbols like `;`, `:`, `.`, `|`, `?`, `!`' etc. and not just `,`